resizer.go

```diff
@@ -68,7 +68,7 @@ func resizer(buf []byte, o Options) ([]byte, error) {
 	}
 
 	// Try to use libjpeg shrink-on-load
-	if imageType == JPEG && shrink >= 2 {
+	if imageType == JPEG && shrink >= 2 && o.StripMetadata {
```

Comment:
I'm afraid this is going to introduce behavioral breaking changes somehow. Did you test this in detail?

Reply:
I tried to understand where the metadata were stripped within libvips but didn't really succeed...